client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -51,51 +51,57 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <div className={classes.inputLabels}>EMAIL ADDRESS</div>
           <TextField
-            id="username"
-            label={<Typography className={classes.label}>Username</Typography>}
+            id="email"
+            placeholder="Your email"
+            // label={<Typography className={classes.label}>Your email</Typography>}
             fullWidth
             margin="normal"
-            InputLabelProps={{
-              shrink: true,
-            }}
+            // InputLabelProps={{
+            //   shrink: true,
+            // }}
             InputProps={{
               classes: { input: classes.inputs },
             }}
-            name="username"
-            autoComplete="username"
+            name="email"
+            autoComplete="email"
             autoFocus
-            helperText={touched.username ? errors.username : ''}
-            error={touched.username && Boolean(errors.username)}
-            value={values.username}
+            helperText={touched.email ? errors.email : ''}
+            error={touched.email && Boolean(errors.email)}
+            value={values.email}
             onChange={handleChange}
           />
+          <div className={classes.inputLabels}>NAME</div>
```

Comment:
Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.
Change `div` to MUI `Box`